### DISCUSSION TITLE
Multiple markdown in table

### DISCUSSION THREAD

**Discussion by vennizemi** [sqlpage/SQLPage#287] [Q&A] [answered]
Hello everyone!

Tell me please, how to properly add a second markdown as a link? (column "A").

Scr 1 : my failed result
![image](https://github.com/lovasoa/SQLpage/assets/162549698/e7dce8d0-1de0-48d2-a47e-dab2fa219833)

Scr 2: code:
![image](https://github.com/lovasoa/SQLpage/assets/162549698/88f4b833-eb35-48d2-9c71-422dd4c85359)

**Comment by lovasoa** [comment, marked as answer]
Hello and welcome to SQLPage!

To pass multiple values to an attribute when using the *dynamic* component, you can use a json array as the attribute value: 

```sql
json_build_object(
  'markdown', json_array('col1', 'col2')
)
```

**Reply by vennizemi** [reply]
Thank you very much! )

**Reply by lovasoa** [reply]
Out of curiosity, what is it that you are building with SQLPage?
